pxtcompiler/emitter/service.ts

```diff
@@ -1145,20 +1170,98 @@ namespace ts.pxtc.service {
             fnName = snakify(fnName).toLowerCase();
 
         const attrs = fn.attributes;
+
+        // TODO(dz)
+        console.log("attrs")
+        console.dir(attrs)
+        function getShadowSymbol(paramName: string): SymbolInfo | null {
+            let shadowBlock = (attrs._shadowOverrides || {})[paramName]
+            if (!shadowBlock)
+                return null
+            const shadowBlockMap: pxt.Map<string> = {
+                // shadow override -> qname
+                // TODO: determine this programatically somehow
+                "minecraftBlockField": "Block",
```

Comment:
add section in pxtarget.json

Reply:
We should be able to enumerate blocks and find ones with matching IDs